.github/workflows/test-pull_request.yaml

```diff
@@ -156,3 +156,74 @@ jobs:
           else
               "./bin/$CONFIGURATION/unittests" --gtest_shuffle
           fi
+
+  build_mac:
+    strategy:
+      matrix:
```

Comment:
`matrix.compiler` is not defined

Reply:
Do we need it? It's always clang on Mac.

Reply:
`matrix.compiler` is used elsewhere in the .yaml.  For convenience its probably best to define it as a matrix member.

Reply:
Updated.